src/components/RangeSlider/tests/RangeSlider.test.tsx

```diff
@@ -38,57 +39,58 @@ describe('<RangeSlider />', () => {
         step: 9,
       };
 
-      const element = mountWithAppProvider(
+      const element = mountWithApp(
         <RangeSlider
           value={[0, 25]}
           {...mockRangeSliderProps}
           {...overrideProps}
         />,
       );
 
-      const {min, max, id, step} = element.find(DualThumb).props();
-
-      expect({
-        min,
-        max,
-        step,
-        id,
-      }).toStrictEqual(overrideProps);
+      expect(element).toContainReactComponent(DualThumb, {
+        ...overrideProps,
+      });
```

Comment:
Suggestion: `(DualThumb, overrideProps)` instead of `(DualThumb, {...overrideProps})` — no need to create a new object here